Frontend/lexica-next-front/src/hooks/useShortcuts.ts

```
@@ -0,0 +1,45 @@
+
+  useEffect(() => {
+    handlers.forEach((h) => {
+      registerShortcut({
+        key: h.key,
+        scope,
+        handler: h.handler,
+      });
+    });
+
+    return () => {
+      handlers.forEach((h) => unregisterShortcut(h.key, scope));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)